powers.c

```diff
@@ -0,0 +1,18 @@
+/*Aldin Aksay*/
+
+#include <stdio.h>
```

Comment:
You didn't handle incorrect input as needed. It went into an infinite loop of printing zeroes. You also didn't incorporate 2^0 = 1.